src/mappings/Coordinator.ts

```diff
@@ -86,3 +90,41 @@ export function updatePoolValues(poolId: string, today: Day): void {
   )
   pool.save()
 }
+
+export function calculate30DayYields(pool: Pool, block: ethereum.Block): Pool {
+  let date = timestampToDate(block.timestamp)
+  let thirtyDaysAgoTimeStamp = date.minus(BigInt.fromI32(secondsInDay * 30))
+  let thirtyDaysAgo = Day.load(thirtyDaysAgoTimeStamp.toString())
+
+  // If this pool is less than 30 days ago, we assume the initial token price is 1.0
+  let thirtyDaysAgoTokenPriceJunior = BigInt.fromI32(1).times(BigInt.fromI32(10).pow(27))
+  let thirtyDaysAgoTokenPriceSenior = BigInt.fromI32(1).times(BigInt.fromI32(10).pow(27))
```

Comment:
Has setting the token price to 1.0 been discussed with the business team? That sounds a bit arbitrary, I guess the better approach is to just calculate the `n` day yield, where `n` is the number of days the pool exists. 

CC @denniswell @lucasvo

Reply:
Discussed with @denniswell, we want to show N/A for now, so I'll make the field nullable.